contentful-data/migrations/dev-export.js

```diff
@@ -0,0 +1,19 @@
+function migrationFunction(migration, context) {
+    const test = migration.createContentType("test");
+    test
+        .displayField("test")
+        .name("test")
+        .description("")
```

Comment:
Should this be blank?

Reply:
for clarity why is the description blank?

Reply:
This is an auto generated file, it just reflects any changes I made in the environments I was migrating. I'd just made a 'test' content model that had no description in the dev environment in contentful and was testing if it would migrate to preprod